frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/DeleteModal/DeleteModal.tsx

```diff
@@ -58,29 +70,35 @@ export const DeleteModal = ({
       }
     >
       <div className={classes.contentWrapper}>
-        <Paragraph size='small' spacing>
-          {t('local_changes.modal_delete_modal_text')}
-        </Paragraph>
-        <Textfield
-          label={t('local_changes.modal_delete_modal_textfield_label')}
-          size='small'
-          value={nameToDelete}
-          onChange={(e) => setNameToDelete(e.target.value)}
-        />
-        <div className={classes.buttonWrapper}>
-          <Button
-            variant='secondary'
-            color='danger'
-            onClick={handleDelete}
-            disabled={appName !== nameToDelete}
-            size='small'
-          >
-            {t('local_changes.modal_confirm_delete_button')}
-          </Button>
-          <Button variant='secondary' onClick={handleClose} size='small'>
-            {t('general.cancel')}
-          </Button>
-        </div>
+        {isLoading ? (
+          <StudioSpinner />
+        ) : (
```

Comment:
Would this make the dialog to resize when the spinner is displayed? What about adding a spinner inside the delete button instead? Indicating that the action is in progress? 😊 What do you think about that?

Reply:
Not too sure if I understand exactly what you mean. Yes, the modal is changing size at the moment, so that should be fixed in a way. 
But do you want to have it inside the button, or to replace the button when the action is in progress? 😄

Reply:
I was thinking inside the button something simillar to this one => https://mui.com/material-ui/react-button/#loading-button

Reply:
Hmm, I see what you mean, but I'm not sure what I think about it. What do you think when you see it here? It will make the button adjust in size horizontally also. 
https://github.com/Altinn/altinn-studio/assets/133344438/3f085805-57c0-48ee-bd5b-aa15c781ebab

Reply:
I think we need to make the spinner `xsmall` and it should have the same color as the button itself? We also need to add some spacing within the loading space to make it look better?

Reply:
Should we disable the input field as well then? 
Can the user still click the button while the spinner is spinning? 
What happens when the user clicks the "cancel" button while the spinner is going?  I don't think we actually cancel the reset operation then, so we would also have to hide/disable the cancel button. 

I think we should follow the same patterns we use elsewhere in the solution, and either:
- only show the spinner, like f.ex. in the versionControl header components, or
- hide the buttons and show the spinner in their place, like in the "create new app" form.

Thoughts @framitdavid?

Reply:
@nkylstad that's a great point! Then I suggest the second opinion to make it consistant with "create new app". We should still disable the input field while loading. 😊

Reply:
Then we keep the context while loading and the modal will keep it size. 👏

Reply:
Should be fixed now 😄